packages/ubuntu_bootstrap/test/storage/manual/test_manual_storage.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:collection/collection.dart';
+import 'package:flutter/cupertino.dart';
```

Comment:
Could we use `material.dart` or `widgets.dart` for consistency?

Reply:
Aah classic accidental auto-import, good catch!